sdk/python/tests/utils/feature_test_utils.py

```diff
@@ -3,17 +3,59 @@
 import tempfile
 import uuid
 from pathlib import Path
-from typing import Iterator, Union
+from typing import Iterator, List, Union
 
 import numpy as np
 import pandas as pd
 import pyarrow
+import pytz
 
 from feast import FeatureService, FeatureStore, FeatureView
 from feast.errors import FeatureViewNotFoundException
 from feast.feature_logging import LOG_DATE_FIELD, LOG_TIMESTAMP_FIELD, REQUEST_ID_FIELD
 from feast.protos.feast.serving.ServingService_pb2 import FieldStatus
 
+"""
+Return latest rows in a dataframe based on join key and entity values.
```

Comment:
nit: move this docstring into `get_latest_rows`?

also add types and return type?

Reply:
fixed